src/current/cockroachcloud/configure-scim-provisioning.md

```diff
@@ -128,27 +146,81 @@ To learn more, refer to [Deprovision a user](https://help.okta.com/en-us/Content
+      1. Find a group to push, and click its name.
+      1. Under **Match result & push action**, click **Create Group** to create a new group in {{ site.data.products.db }} and link the Okta group to it. If a group exists in {{ site.data.products.db }} with the same name, click **Link Group**.
    1. To unlink a linked group, select it, then click **Unlink pushed group**.
    1. If you disabled **Push group memberships immediately**, click **Push Now**.
 
-1. To disable Group Push, click **Deactivate Group Push**.
+### Disable Group Push
+
+To disable Group Push, click **Deactivate Group Push**. You can optionally delete the group from {{ site.data.products.db }}. If you choose not to delete it, the group's membership stops being updated by the SCIM app integration, but the group's assigned roles continue to be applied to its members and can be modified.
+
+## Manage groups in {{ site.data.products.db }}
+
+### View a group's details
 
+To view details about a group:
 
-For more information and troubleshooting, refer to [Manage Group Push](https://help.okta.com/en-us/Content/Topics/users-groups-profiles/usgp-group-push-main.htm) in the Okta documentation.
+1. In {{ site.data.products.db }} Console, click **Access Management > Groups**.
+1. In a group's row, click the three-dots **Action** button, then select any of **View Members**, **View Parent Groups**, or **View Child Groups**.
+
+### Manage a group's roles
+
+Within {{ site.data.products.db }}, you can grant [roles](authorization.html#organization-user-roles) to a pushed group, and those roles are automatically granted to the group's members who have accounts in {{ site.data.products.db }}.
+
+- When you begin pushing a group whose members already exist in {{ site.data.products.db }}, those members are granted the group's roles, in addition to roles explicitly granted to them.
+- When the group's membership changes in your IdP, those changes are synchronized with the group in {{ site.data.products.db }}. If a {{ site.data.products.db }} account is added to or removed from the group in your IdP, they gain or lose roles granted to the group in {{ site.data.products.db }}.
+
+This section shows how to view and manage a group's roles in the {{ site.data.products.db }} Console. You can also use the [{{ site.data.products.db }} SCIM API](https://cockroachlabs.com/docs/api/cloud/v1.html#tag--SCIM).
+
+1. In {{ site.data.products.db }} Console, click **Access Management > Groups**.
+1. In a group's row, click the three-dots **Action** button, then click **Edit Roles**.
+1. The group's granted roles are shown. Add or remove roles, then click **Confirm**.
+
+      {{site.data.alerts.callout_info}}
+      The fields for inherited roles are read-only, because inherited roles cannot be edited directly. Instead, you must either remove the role from the parent group from which it is inherited, or remove the child group from the parent group.
+      {{site.data.alerts.end}}
+
+### Get a group's ID
```

Comment:
And make this header as "Manage a group's roles using API"? 

And then add the link to the Roles API - https://www.cockroachlabs.com/docs/api/cloud/v1#tag--Role-Management and TF provider - https://registry.terraform.io/providers/cockroachdb/cockroach/latest/docs/resources/user_role_grants after the three links you already have to get the Group's ID.

Reply:
Instead of adding another section, I reworked this to use tabs, one for the Console and one for the API. Added your suggested links to the API tab.